Readme.md

```diff
@@ -173,6 +173,32 @@ AppConfig[:aeon_fulfillment] = {
   this setting is not specified in the settings for the repository, no Aeon
   site code will be sent.
 
+- **:hide\_button\_for\_access\_restriction\_types**. This setting allows
+  the request button to be hidden for any records that have any of the
+  listed local access restriction types. The value of this config item
+  should be an array of restriction types, for example:
+      `:hide_button_for_access_restriction_types => ['RestrictedSpecColl']`
+  By default no restriction types are hidden.
+
+- **:site**. This setting specifies the site code for a repository.
```

Comment:
Good morning James, is this meant to allow sites to send requests to a specific Aeon site? If so, this was added on Friday as the `:aeon_site_code` repo setting.

Reply:
Hi @AustinTSchaffer - I might be missing something, but, as noted above, I'm sending it as `Site` which seems to drop it directly in the `Site` field in Aeon, rather than relying on an openurl mapping.